jakarta-xml-binding/bin/run-tck.sh

```diff
@@ -201,7 +209,7 @@ jck.env.jaxb.xsd_compiler.run.compilerWrapperClass=com.sun.jaxb_tck.lib.SchemaCo
 jck.env.jaxb.xsd_compiler.skipValidationOptional=Yes
 jck.env.testPlatform.local=Yes
 jck.env.testPlatform.multiJVM=No
-jck.excludeList.customFiles=${TCK_HOME}/lib/jaxb_tck40.jtx
+jck.excludeList.customFiles=${TCK_HOME}/lib/jaxb_tck40.jtx${ADDITIONAL_EXCLUDE_FILE}
```

Comment:
Do we need a separator after the first .jtx file?

Reply:
like ":" or does the new line work?

Reply:
@scottmarlow I'd asked James that question and he said it seemed to need to be \n